x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/kibana/disable_internal_collection_instructions.js

```diff
@@ -0,0 +1,221 @@
+          values={{
+            secondsSinceLastInternalCollectionLabel,
+          }}
+        />);
+      }
+
+      status = (
+        <Fragment>
+          <EuiSpacer size="m"/>
+          <EuiCallOut
+            size="s"
+            color="warning"
+            title={i18n.translate('xpack.monitoring.metricbeatMigration.kibanaInstructions.partiallyMigratedStatusTitle',
+              {
+                defaultMessage: `We still see data coming from internal collection of Kibana.
+                Note that it can take up to {secondsAgo} seconds to detect.
+                We will continuously check every {timePeriod} seconds in the background.`,
+                values: {
+                  secondsAgo: meta.secondsAgo,
+                  timePeriod: autoCheckIntervalInMs / 1000,
+                }
+              }
+            )}
+          >
+            <EuiText>{lastInternallyCollectedMessage}</EuiText>
+          </EuiCallOut>
+        </Fragment>
+      );
+    }
+
+    let buttonLabel;
+    if (checkingMigrationStatus) {
+      buttonLabel = i18n.translate(
+        'xpack.monitoring.metricbeatMigration.kibanaInstructions.disable_internal_collection.checkingStatusButtonLabel',
+        {
+          defaultMessage: 'Checking...'
+        }
+      );
+    } else {
+      buttonLabel = i18n.translate(
+        'xpack.monitoring.metricbeatMigration.kibanaInstructions.disable_internal_collection.checkStatusButtonLabel',
+        {
+          defaultMessage: 'Check'
+        }
+      );
+    }
+
+    migrationStatusStep = {
+      title: statusTitle,
+      status: 'incomplete',
+      children: (
+        <Fragment>
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem>
+              <EuiText>
+                <p>
+                  {i18n.translate(
+                    'xpack.monitoring.metricbeatMigration.kibanaInstructions.disable_internal_collection.statusDescription',
+                    {
+                      defaultMessage: 'Check that no documents are coming from internal collection.'
+                    }
+                  )}
+                </p>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiButton onClick={checkForMigrationStatus} isDisabled={checkingMigrationStatus}>
+                {buttonLabel}
+              </EuiButton>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          {status}
+        </Fragment>
+      )
+    };
+  }
+  else {
+    migrationStatusStep = {
+      title: statusTitle,
+      status: 'complete',
+      children: (
+        <EuiCallOut
+          size="s"
+          color="success"
+          title={i18n.translate(
+            'xpack.monitoring.metricbeatMigration.kibanaInstructions.disable_internal_collection.fullyMigratedStatusTitle',
+            {
+              defaultMessage: 'Congratulations! We are not seeing any documents from internal collection. Migration complete!'
```

Comment:
```suggestion
              defaultMessage: 'Migration complete!'
```

Then put `We are not seeing any documents from internal collection.` as the child wrapped in a `<p>` tag.

Reply:
<img width="941" alt="Screen Shot 2019-06-14 at 10 05 46 AM" src="https://user-images.githubusercontent.com/56682/59514949-03b0e100-8e8c-11e9-821b-3e0360ff08f2.png">

Reply:
Wrong screenshot?

Reply:
Yes, my bad

<img width="869" alt="Screen Shot 2019-06-14 at 10 06 40 AM" src="https://user-images.githubusercontent.com/56682/59515001-23e0a000-8e8c-11e9-83a5-b6cc2bf56392.png">

Reply:
Yes, my bad
<img width="919" alt="Screen Shot 2019-06-14 at 12 45 58 PM" src="https://user-images.githubusercontent.com/56682/59524619-601efb00-8ea2-11e9-8e54-4f44a7fcd8db.png">